computations/x1_16_chabauty.m

```diff
@@ -33,8 +43,10 @@ function EllipticChabauty(C, g, hk, HktoA)
   assert Evaluate(f, L.1) eq 0;
   AtoL := hom< A -> L | L.1>;
 
-  gamma := Norm(AtoL(HktoA(hk)));
+  // For some gamma, we have that a 2-cover hk covers the elliptic curve E:y^2=gamma * g.
```

Comment:
This could theoretically be a pointless genus 1 curve if the degree of g is even.

Reply:
done